CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
-## 1.20.0 (Unreleased)
+## 1.21.0 (Unreleased)
+
+## 1.20.0 (September 16, 2020)
+
+BUG FIXES:
+* `resource/cce_cluster_v3` and `data_source/cce_cluster_v3`: Update outdated docs ([#614](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/614))
+* `resource/lb_listener_v2`: Update outdated docs ([#615](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/615))
+
+ENHANCEMENTS:
+* **New data source:** `data_source/opentelekomcloud_identity_credential_v3` Add identity_credential_v3 (permanent AK/SK) ([#613](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/613))
```

Comment:
The description should be separated from the resource name with `:`. But generally, new res or data doesn't have descriptions.